src/geometries/primitives/parametrizedcurve.jl

```diff
@@ -0,0 +1,48 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    ParametrizedCurve(func, ab = (0.0, 1.0))
+
+A parametrized curve is a curve defined by a function `func` that maps a parameter `t` to a `Point` in space.
+The parameter `t` is defined in the interval `[a, b]`. The curve can only be evaluated for `t` in the
+interval `[0, 1]`.
```

Comment:
```suggestion
A curve defined by a function `func` that maps a parameter `t` in the real line to a
`Point` in space. The parameter `range` can be specified, and defaults to `[0,1]` otherwise.
```

Reply:
Reading that as a user, I would assume I can evaluate my curve in the interval [a, b] if I use a custom `range`, but that's not true. I would like to make clear that the curve can only be evaluated in the default range [0, 1].